src/Food.js

```diff
@@ -1,24 +1,48 @@
-import { useState } from "react";
-
 function Food(props) {
-    const removeFood = () => {
+
+    const newAddLikes = () => {
+        var likeCopy = [...props.likeCount];
+        likeCopy[props.index] = likeCopy[props.index] + 1;
+        props.setLikeCount(likeCopy);
+    }
+
+    const removeFood = () => {    
         const foodCopy = [...props.foodList.splice(0, props.index), ...props.foodList.splice(props.index + 1)];
-        const likeCopy = [...props.likes.splice(0, props.index), ...props.likes.splice(props.index + 1)];
+        if (props.index === foodCopy.length) {
+            props.setCurrIndex(0);
+        }
         props.setFoodList(foodCopy);
-        props.setLikes(likeCopy);
+
+        const likeCopy = [...props.likeCount.splice(0, props.index), ...props.likeCount.splice(props.index + 1)];
+        props.setLikeCount(likeCopy);
     }
 
-    const addLikes = () => {
-        var obj = {...props.likes};
-        obj[props.index] = 'a';
-        props.setLikes({obj});
+    const nextFood = () => {
+        if (props.index < props.foodList.length - 1) {
+            props.setCurrIndex(props.index + 1);
+        } else {
+            props.setCurrIndex(0);
+        }
```

Comment:
nit: can you think of a way to use modulo (`%`) to condense this into a singular line?

lmk if you want me to explain what `%` is!